src/nft/utils/listNfts.ts

```diff
@@ -62,7 +62,7 @@ const getConsiderationItems = (
   creatorFee?: ConsiderationInputItem
 } => {
   const openSeaBasisPoints = OPENSEA_DEFAULT_FEE * INVERSE_BASIS_POINTS
-  const creatorFeeBasisPoints = asset.basisPoints
+  const creatorFeeBasisPoints = asset?.basisPoints ?? 0
```

Comment:
i feel like we non-null cast a lot. Should we let those fields be null until we /need/ to display them?

Reply:
This is where it's needed to calculate creator royalties, in this case if the value is undefined we can assume that no royalty amount is set